assignments/pr_review_tutorial/ingram/buggy_function.py

```diff
@@ -30,3 +30,4 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
 
     format_string = '{}:{}:{:.' + str(decimals) + 'f}'
     return format_string.format(hours, minutes, seconds)
+#idk whats wrong xD
```

Comment:
I would suggest removing this comment. 

(the bug in the code was e.g. that the conversion between hours and degrees is 15 rather than 24/180. Ideally it would also check whether the angle input is within 0 and 360 degrees, and also there is no need to use `math.floor()` to decide whether a number is an integer. No one else fixed any of these, so I would suggest just to remove the comment from above.)